src/effects/builtin/whitenoiseeffect.cpp

```diff
@@ -0,0 +1,65 @@
+#include "whitenoiseeffect.h"
+
+#include <QtDebug>
+#include <QRandomGenerator>
+
+// static
+QString NoiseEffect::getId() {
+    return "org.mixxx.effects.noise";
+}
+
+// static
+EffectManifestPointer NoiseEffect::getManifest() {
+    EffectManifestPointer pManifest(new EffectManifest());
+    pManifest->setId(getId());
+    pManifest->setName(QObject::tr("White Noise"));
+    pManifest->setAuthor("The Mixxx Team");
+    pManifest->setVersion("1.0");
+    pManifest->setDescription(QObject::tr("Creates active white noise sound"));
+
+    // This is loudness parameter
+    EffectManifestParameterPointer intensity = pManifest->addParameter();
+    intensity->setId("Loudness");
+    intensity->setName(QObject::tr("Loudness"));
+    intensity->setDescription(QObject::tr("Set the loudness which makes the noise louder or more quiet"));
+    intensity->setControlHint(EffectManifestParameter::ControlHint::KNOB_LOGARITHMIC);
+    intensity->setSemanticHint(EffectManifestParameter::SemanticHint::UNKNOWN);
+    intensity->setUnitsHint(EffectManifestParameter::UnitsHint::UNKNOWN);
+    intensity->setDefaultLinkType(EffectManifestParameter::LinkType::LINKED);
+    intensity->setMinimum(0);
+    intensity->setDefault(0);
+    intensity->setMaximum(1);
+
+    return pManifest;
+}
+
+NoiseEffect::NoiseEffect(EngineEffect* pEffect)
+        : m_pLoudnessParameter(pEffect->getParameterById("Loudness")) {
+}
+
+NoiseEffect::~NoiseEffect() {
+}
+
+void NoiseEffect::processChannel(
+        const ChannelHandle& handle,
+        NoiseGroupState* pGroupState,
+        const CSAMPLE* pInput,
+        CSAMPLE* pOutput,
+        const mixxx::EngineParameters& bufferParameters,
+        const EffectEnableState enableState,
+        const GroupFeatureState& groupFeatures) {
+    Q_UNUSED(handle); /*The Q_UNUSED() macro can remove the warning generated by the compiler due to the unused parameter,
```

Comment:
I don't think that comment is needed here. Also, we don't use block comments in our code anymore, because they break if you want to comment out a larger block of code quickly.